### DISCUSSION TITLE
看了这么多呼吁，怎么就没有呼吁严厉处罚烂尾楼房地产商的呢

### DISCUSSION THREAD

**Discussion by lalalabbbb** [WeNeedHome/SummaryOfLoanSuspension#106] [General]
如题，看了这么多呼吁，怎么就没有呼吁严厉处罚烂尾楼房地产商的呢

**Comment by AntEffect** [comment]
堂下何人，状告本官呐？

**Comment by fuqian95** [comment]
因为烂尾楼的开发商基本都没钱了，你再怎么要求处罚，它就是没钱了，盖不了， 没那个能力。

**Reply by lalalabbbb** [reply]
把人抓去关

**Reply by MarquessZhonggua** [reply]
人家可以先转移资产，再申请破产的。

**Reply by DDFighting** [reply]
曲婉婷他妈都关进去多少年了，钱追回来了么？

**Comment by zhaokai5520** [comment]
企业都是有限责任公司，

**Comment by jiujilalala** [comment]
不少开发商是当地的白手套，没有自己人抓自己人的

**Reply by jiujilalala** [reply]
你不会觉得大公司的区域总城市总项目总和当地政府只是点头之交吧

**Reply by jiujilalala** [reply]
你在我的地盘有项目，让我的亲戚子女作为工程项目的一环赚点费用不过分吧

**Comment by xcb425** [comment]
公司都是套皮的,法人都是买的身份证.当事人早跑国外享福去了,怎么惩罚?最多抓几个高级打工人而已.

**Comment by MarquessZhonggua** [comment]
参考刘某周事件，开发商可以申请破产，全身而退。

**Comment by jokerdrake** [comment]
他们只赚15%，工具狗而已，站在后面赚70%以上的那个敲骨吸髓的大家都懂的。谁敢责怪真正的那帮人啊。

**Comment by NearX** [comment]
开发商只是个背锅的。。。 你以为它们怎么自己想挪资金就挪的？

**Comment by ZhangJiajunK** [comment]
一个楼盘成立一个公司，公司是有限责任的，钱被违规挪用，导致资金链断裂，公司可以申请破产的，破产仅以该公司的净资产清偿债务。对就是单个楼盘的这个公司承担，跟集团内的其他房产公司没有丝毫关系，跟背后的股东大老板更没有关系。这帮人赚钱了就赶快分红，靠着借债杠杆开发房产，烂尾，破产对他们可以说没多大影响。

**Comment by C3604** [comment]
当前法律制度不完善，所以没有有效的法律支持

**Comment by luzhiy123** [comment]
世界上出现第一个抢劫犯的时候，在审判时法官判处抢劫无罪

然后出去之后是无忌惮的抢劫，其他人也有样学样。

后面法律上加了个抢劫有罪，大多都消停了一会儿，但是仍有人继续抢劫也没人被判。大家都发现，没人再管这些事情了，就更变本加厉起来了。抢劫犯多了，抢劫的频率也高了，这些是谁的责任？

第一次抢劫时第一个抢劫犯的责任，后续的案件，这个法官和抢劫犯谁的责任更大？全国上下都是抢劫犯是谁的过？

**Comment by Jackwew** [comment]
是否可以把烂尾的开发商也记入失信人员黑名单

**Reply by WallStreetHu** [reply]
没用的，钱早转移完了，注册资金一个亿，卷了几个亿，玩破产、搞失信黑名单，他也不亏啊。

**Comment by dengwubogit** [comment]
他们脸都不要了,讨伐他们能有个鸟用,考验银行公信力的时候到了

**Reply by middleHallo** [reply]
给你个红马自己体会

**Comment by xiaohao587** [comment]
维尼、银行、开发商都是局中人，只有你个买房才是局外人，你还说追开发商的责任？

**Comment by jiang443** [comment]
能说清楚开发商犯了什么罪吗。。
